flask_whooshee.py

```diff
@@ -265,9 +267,15 @@ def create_index(self, wh):
                 os.makedirs(index_path)
             index = whoosh.index.create_in(index_path, wh.schema)
         wh.index = index
-
-    def on_commit(self, app, changes):
-        """Method that gets connected to flask_sqlalchemy.models_committed, where it serves
+    def after_insert(self, mapper, connection, target):
```

Comment:
Can you please put newlines between the newly defined methods? Just so the code reads better.